input.html

```diff
@@ -4,15 +4,36 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz-App | Create</title>
   </head>
   <body>
-    <nav>
-      <a href="./index.html">Homepage</a>
-      <a href="./bookmark.html">Bookmark</a>
-      <a href="./input.html">Input</a>
-      <a href="./profile.html">Profile</a>
-    </nav>
-    <p>Input</p>
+    <header>
+      <h1>Create</h1>
+    </header>
+    <main>
+      <form action="">
+        <h2><label for="">Add question</label></h2>
+        <textarea id="question" name="question" rows="4" cols="30"></textarea>
+        <h2><label for="">Add answer</label></h2>
+        <textarea id="question" name="tags" rows="4" cols="30"></textarea>
+        <h2><label for="">Add tags</label></h2>
+        <textarea
+          id="tags"
+          name="tags"
+          rows="1"
+          cols="30"
+          placeholder="Add tags divided by comma."
+        ></textarea>
+        <input type="submit" value="Submit" />
+      </form>
```

Comment:
After this morning's session (HTML Forms), you are going to slightly adapt this part, but it is a really good start!

Reply:
Hey @PReichetanz , could you please take another look at the updated form element? Thank you :)